Release 16-10-2024 (#231)
LiskHQ/lisk-docs: examples.md

# Example Blockchain Applications

## Hello World App

> Check out the full code example for the [Hello World App on Github](https://github.com/LiskHQ/lisk-sdk-test-app/tree/development/hello_world).

> A simple App, showcasing a minimal setup of a blockchain application with 1 custom transaction type: the "Hello" transaction.

Reviewer: ```suggestion
A simple App, showcasing a minimal setup of a blockchain application with 1 custom transaction type: the "HelloWorld" transaction.
```

Author: imho it should be clear enough. See my comment here https://github.com/LiskHQ/lisk-docs/pull/247#discussion_r290220964

## Cashback App

> Check out the full code example for the [code example on Github](https://github.com/LiskHQ/lisk-sdk-test-app).

> A simple application which is rewarding it's users for sending tokens. 

Reviewer: ```suggestion
A simple application which rewards its users for sending tokens. 
```

To achieve this, a custom transaction type `CashbackTransaction` is created.

> It extends the already existing transaction type `TransferTransaction`.

Reviewer: ```suggestion
It extends the pre-existing transaction type `TransferTransaction`.
```

> The here Cashback transaction type return half of the amount of tokens that are transferred back to the senders account.

Reviewer: ```suggestion
Here the Cashback transaction type pays out a 50% bonus reward to the sender of any Cashback transaction type.
```
